src/main/java/com/plugish/woominecraft/WooMinecraft.java

```diff
@@ -177,8 +191,12 @@ boolean check() throws Exception {
 
 			// Walk over all commands and run them at the next available tick.
 			for ( String command : order.getCommands() ) {
-				BukkitScheduler scheduler = Bukkit.getServer().getScheduler();
-				scheduler.scheduleSyncDelayedTask( instance, () -> Bukkit.getServer().dispatchCommand( Bukkit.getServer().getConsoleSender(), command ), 20L );
+				//Auth player against Mojang api
+				if (AuCh(player)) {

```

Comment:
Lets switch this to return early instead of dropping the code within a guard clause. Part of the age-old `return early and often`